vnext/PropertySheets/CppNuGetReferences.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file doesn't and likely never will define actual targets.
Shouldn't we use the `props` extension?